tests/functional/disaster-recovery/regional-dr/test_dryrun_failover.py

```
@@ -0,0 +1,632 @@
+        dr_workload,
+        discovered_apps_dr_workload,
+    ):
+        """
+        Test dryRun failover followed by abort across all three DR pre-states:
+        Deployed, FailedOver, and Relocated — for both ApplicationSet and
+        DiscoveredApps workloads simultaneously.
+
+        Verifies cluster_b is cleaned up and each workload remains healthy
+        on cluster_a after abort.
+        """
+        workload_info, cluster_a, cluster_b, scheduling_interval, wait_time = (
```

Ruff (0.15.21)

[warning] Unpacked variable `wait_time` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)